app/forms/steps/income/partner/employment_income_form.rb

```diff
@@ -0,0 +1,54 @@
+module Steps
+  module Income
+    module Partner
+      class EmploymentIncomeForm < Steps::BaseFormObject
+        attribute :amount, :pence
```

Comment:
Adding employment payments is breaking the view of income payments 
<img width="1129" alt="Screenshot 2024-06-21 at 16 17 39" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/51047911/aa4fbc5e-bc24-4f22-b589-817675dacac2">

Reply:
Will handle in separate bug ticket